subprojects/gradle/buildchecks/src/main/kotlin/com/avito/android/plugin/build_param_check/BuildParamCheckPlugin.kt

```diff
@@ -236,14 +240,14 @@ open class BuildParamCheckPlugin : Plugin<Project> {
         }
     }
 
-    private fun printBuildEnvironment(project: Project) {
+    private fun printBuildEnvironment(project: Project, logger: Logger) {
         val isBuildCachingEnabled = project.gradle.startParameter.isBuildCacheEnabled
         val minSdk = project.getOptionalStringProperty("minSdk")
         val kaptBuildCache: Boolean = project.getBooleanProperty("kaptBuildCache")
         val kaptMapDiagnosticLocations = project.getBooleanProperty("kaptMapDiagnosticLocations")
         val javaIncrementalCompilation = project.getBooleanProperty("javaIncrementalCompilation")
 
-        project.ciLogger.info(
+        logger.info(
```

Comment:
debug

Reply:
it says "config information for project" as first line, looks like clear info level for me

Reply:
https://github.com/avito-tech/avito-android/discussions/698 Created a discussion about log levels in our project

Reply:
Leave a comment